Content.Server/Emp/EmpSystem.cs

```diff
@@ -31,7 +31,7 @@ public void EmpPulse(MapCoordinates coordinates, float range, float energyConsum
     {
         foreach (var uid in _lookup.GetEntitiesInRange(coordinates, range))
         {
-            var ev = new EmpPulseEvent(energyConsumption, false, false);
+            var ev = new EmpPulseEvent(energyConsumption, false, false, TimeSpan.FromSeconds(duration));
```

Comment:
This should be a readonly struct and outside of the loop.

Reply:
The event is by ref, doesn't a new one need to be made for each entity to avoid them essentially sharing responses? 
If entity A responds to the event with a positive, and entity B doesn't respond at all, entity A and B will both be set to positive, correct?